.github/workflows/test-packages.yml

```
@@ -10,44 +10,396 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  CXX: g++-12
+  METEOR_HOSTED_CI: "true"
+  METEOR_MODERN: true
+  CXX: g++-12
```

actionlint (1.7.12)

[error] key "CXX" is duplicated in "env" section. previously defined at line:14,col:3. note that this key is case insensitive

(syntax-check)